source/unit/Fixture.ooc

```diff
@@ -21,9 +21,25 @@ import Constraints
 
 Fixture: abstract class {
 	totalTime: static Double
+	exitHandlerRegistered: static Bool
+	failureNames: static VectorList<String>
 	name: String
 	tests := VectorList<Test> new()
-	init: func (=name)
+	printFailures: static func {
+		if (This failureNames && (This failureNames count > 0)) {
+			"Failed tests: %i [" format(This failureNames count) print()
+			for (i in 0 .. This failureNames count - 1)
+				(This failureNames[i] + ", ") print()
+			This failureNames[This failureNames count - 1] print()
+			"]" println()
```

Comment:
If we're changing this anyhow, I do prefer `printfln(...)` over `format(...) println()`